### DISCUSSION TITLE
False miss indicator when I didn't miss anything (no combobreak)

### DISCUSSION THREAD

**Discussion by hwabis** [ppy/osu#17394] [Q&A] [answered]
https://user-images.githubusercontent.com/62194990/159352494-f8a637d1-1529-44d2-a4aa-756d1de000c7.mp4

The miss indicator appears even though I didn't miss anything. The indicator only appears after I watch that part of the replay, seek back, then watch it again. The first watch-through is fine (no indicator). This happens consistently; whenever I exit the replay and watch it again, the same behavior occurs. I don't think it's a skin-specific issue either, because this occurs with another one of my skins.

I can provide a replay if requested (I can't upload osr to here directly)

**Comment by bdach** [comment]
please attach the replay in a zip archive.

**Reply by hwabis** [reply]
[Erasome playing Yooh - Ice Angel (ktgster) [Saint].zip](https://github.com/ppy/osu/files/8318930/Erasome.playing.Yooh.-.Ice.Angel.ktgster.Saint.zip)

This is the map: https://osu.ppy.sh/beatmapsets/297933#osu/668662

**Reply by peppy** [reply]
I can't see this issue when watching the replay.

Can you reproduce this every time?

**Reply by hwabis** [reply]
Yes, I just re-downloaded my replay and it's still happening every time. Also make sure you're using a skin with a clear miss indicator (I find it hard to see with the default "osu! (triangles)" skin).

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/17413
